api/query/cache/index/index.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package index
+
+import (
+	"errors"
+	"sort"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+var (
+	errNoRuns = errors.New("No runs")
+	errNilRun = errors.New("Test run is nil")
+)
+
+type byTimeStart []*shared.TestRun
```

Comment:
nit: Dump this in `shared` as `type ByTimeStart TestRuns`

Reply:
Turns out `shared.TestRuns` orders by `TimeStart` by default. I'll just use that.